src/hooks/useDrag.tsx

```diff
@@ -184,6 +184,38 @@ export const useDrag: OnDragEndResponseWrapper = (
   return updateHomepageState(result, homepageState)
 }
 
+type UpdateHomepageType =
+  | "editor"
+  | "dropdownelem"
+  | "highlight"
+  | "announcement"
+  | `textCardItem-${number}`
+
+const isUpdateHomepageType = (
+  value: any
```

Comment:
nit: chaneg to `unknown`

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/1527/commits/8e0b01728961af0602a429a857a7f8648f961103